src/api/config/routes.rb

```diff
@@ -309,7 +309,7 @@ def self.public_or_about_path?(request)
       post 'request/add_role_request' => :add_role_request
       get 'request/set_bugowner_request_dialog' => :set_bugowner_request_dialog
       post 'request/set_bugowner_request' => :set_bugowner_request
-      get 'request/change_devel_request_dialog' => :change_devel_request_dialog
+      get 'request/change_devel_request_dialog/:project/:package' => :change_devel_request_dialog, as: :change_devel_request_dialog, constraints: cons
```

Comment:
I am not sure if we should add names to all the routes... I am able to remember Rails defaults... but those names you always need to look for them... I would say we don't gain anything by introducing them :thinking: 

And if you want to introduce it them I'll squash this commit and the next one.